docs/_includes/generated/task_manager_configuration.html

```diff
@@ -84,13 +84,13 @@
         </tr>
         <tr>
             <td><h5>taskmanager.memory.segment-size</h5></td>
-            <td style="word-wrap: break-word;">32768</td>
-            <td>Size of memory buffers used by the network stack and the memory manager (in bytes).</td>
+            <td style="word-wrap: break-word;">"32768"</td>
+            <td>Size of memory buffers used by the network stack and the memory manager.</td>
         </tr>
         <tr>
             <td><h5>taskmanager.memory.size</h5></td>
-            <td style="word-wrap: break-word;">-1</td>
-            <td>Amount of memory to be allocated by the task manager's memory manager (in megabytes). If not set, a relative fraction will be allocated.</td>
+            <td style="word-wrap: break-word;">"-1"</td>
```

Comment:
looking at the [parser](https://github.com/yanghua/flink/blob/39fd2efb66bd6fcac3f86e953729831a49bc7709/flink-core/src/main/java/org/apache/flink/configuration/MemorySize.java#L191) it appears that `MemorySize` does not support negative values (rightfully so). We may have to change the default to 0.

Reply:
currently, every place we use the value of config item `taskmanager.memory.size`, will compare with the default value "-1", if yes, will not use the parser, so there is no problem now. But I think the suggestion from @zentol  is good. What's your opinion? @StephanEwen